crates/recent_projects/src/ssh_connections.rs

```diff
@@ -351,9 +351,18 @@ impl remote::SshClientDelegate for SshClientDelegate {
         rx
     }
 
-    fn remote_server_binary_path(&self, cx: &mut AsyncAppContext) -> Result<PathBuf> {
+    fn remote_server_binary_path(
+        &self,
+        platform: SshPlatform,
+        cx: &mut AsyncAppContext,
+    ) -> Result<PathBuf> {
         let release_channel = cx.update(|cx| ReleaseChannel::global(cx))?;
-        Ok(format!(".local/zed-remote-server-{}", release_channel.dev_name()).into())
+        Ok(format!(
+            ".local/libexec/zed-remote-server-{}-{}",
```

Comment:
Shouldn't this be in `support_dir()`, roughly `$XDG_DATA_HOME` or `$HOME/.local/share`on Linux? I think putting it straight into `.local` was already incorrect.